### DISCUSSION TITLE
Fail to spawn forcetoslot="true" ammobox inside creature inventory in multiplayer

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#14938] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

I'm using forcetoslot="true" to put coilgunammobox inside creature inventory as loot
It works fine in singleplayer but in multiplayer the ammo box won't spawn

### Reproduction steps

```xml
<Inventory Slots="Any, Any" AccessibleWhenAlive="True" Commonness="1">
  <Item Identifier="coilgunammobox" forcetoslot="true" />
</Inventory>
```

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
No issues found, closing discussion as resolved.
